### ISSUE TITLE
Include plugin health score in Pipeline steps reference

### ISSUE THREAD

**Issue by MarkEWaite** [jenkins-infra/plugin-health-scoring#181]
I think that users of the [Pipeline Steps reference pages](https://www.jenkins.io/doc/pipeline/steps/) (like [docker compose build step](https://www.jenkins.io/doc/pipeline/steps/docker-compose-build-step/)) would benefit if the top of the page included the plugin health score as either a badge or a visible number.  That would help users of the specific Pipeline steps understand if they are using a plugin that is unmaintained or otherwise unhealthy.

The [docker compose build step](https://plugin-health.jenkins.io/scores/docker-compose-build-step) (as an example) has a surprisingly high score (96) for a plugin that has had only one release and that single release was 5 years ago with documentation from the Jenkins wiki export.  Likely that we need more components in the health score in order to more accurately reflect the lower score that type of plugin should receive.

**Comment by alecharp** [comment]
> The [docker compose build step](https://plugin-health.jenkins.io/scores/docker-compose-build-step) (as an example) has a surprisingly high score (96) for a plugin that has had only one release and that single release was 5 years ago with documentation from the Jenkins wiki export. Likely that we need more components in the health score in order to more accurately reflect the lower score that type of plugin should receive.

I agree. We need to refine the scoring process or create a new one now that we have the Jenkins Core version requirement, the installation count, etc. cc @jleon33

**Comment by Jagrutiti** [comment]
Is this a beginner-friendly issue?

**Comment by MarkEWaite** [comment]
> Is this a beginner-friendly issue?

I don't think this is a good first issue yet.  Once the health scores can be downloaded from the plugin health scoring system, it might be a good first issue.

The majority of the changes to implement this will be in the [Pipeline steps doc generator](https://github.com/jenkins-infra/pipeline-steps-doc-generator) and possibly in the [Pipeline metadata utils](https://github.com/jenkins-infra/pipeline-metadata-utils), not in the plugin health score repository.  Those tools will need to be enhanced to read the plugin health score data and insert the scores into the page layout of each of the pipeline steps pages in a way that is visually pleasing.

**Comment by Jagrutiti** [comment]
Could you tell me a beginner-friendly issue in this repository?

**Comment by MarkEWaite** [comment]
I don't see any issues in the repository currently that I would consider good first issues.
